src/samples/DistanceImage.hpp

```diff
@@ -79,16 +79,15 @@ namespace samples
 	template <class Scalar_>
 	bool getScenePoint( size_t x, size_t y, Eigen::Matrix<Scalar_,3,1>& point ) const
 	{
-	    point = Eigen::Matrix<Scalar_,3,1>( (x*scale_x)+center_x, (y*scale_y)+center_y, 1.0 );
```

Comment:
This would typically be a case for a unit test. "Don't write the output arg if input is wrong" is a common bug.